lib/jekyll/commands/post.rb

```diff
@@ -64,8 +64,18 @@ def _time_stamp
         end
 
         def content(custom_front_matter = {})
+          if jekyll_compose_config && jekyll_compose_config["post_default_front_matter"]
+            custom_front_matter.merge!(jekyll_compose_config["post_default_front_matter"])
+          end
+
           super({ "date" => _time_stamp }.merge(custom_front_matter))
         end
+
+        private
+
+        def jekyll_compose_config
+          @jekyll_compose_config ||= Jekyll.configuration["jekyll_compose"]
```

Comment:
Let's default the config to a hash so that we can avoid a `nil` check in method `#content` above:
```ruby
# Returns a Hash
def compose_config
  @compose_config ||= Jekyll.configuration["jekyll_compose"] || {}
end
```

Reply:
This has been adressed in #76